md/deploy-bundle.md

```diff
@@ -75,11 +75,11 @@ use the driver provided by your RDBMS vendor
 ## Wildfly installation
 
 ### Install Bonita BPM Platform in Wildfly
-Copy all files and directories from DEPLOY_ZIP_HOME/wildfly-10.1.0.Final to your Wildfly root directory (WILDFLY_HOME).
+Copy all files and directories from DEPLOY_ZIP_HOME/wildfly-10.1.0.Final/server to your Wildfly root directory (WILDFLY_HOME).
```

Comment:
Actually Wildfly distrib has a directory named `server` so we should do `cp -r DEPLOY_ZIP_HOME/wildfly-10.1.0.Final/server/* WILDFLY_HOME/server/*`
I guess this equivalent to `cp -r DEPLOY_ZIP_HOME/wildfly-10.1.0.Final/* WILDFLY_HOME/*` :-)